resources/services/memorystore/gen.hcl

```diff
@@ -0,0 +1,214 @@
+    path = "github.com/cloudquery/cq-provider-gcp/client.DeleteProjectFilter"
+  }
+
+  options {
+    primary_keys = [
+      "project_id",
+      "id"
+    ]
+  }
+
+  ignore_columns_in_tests = [
+    "suspension_reasons",
+    "maintenance_policy",
+    "maintenance_schedule",
+    "redis_configs",
+  ]
+
+  userDefinedColumn "project_id" {
+    type        = "string"
+    description = "GCP Project ID of the resource"
+    resolver "resolveResourceProject" {
+      path = "github.com/cloudquery/cq-provider-gcp/client.ResolveProject"
+    }
+  }
+
+  userDefinedColumn "id" {
+    type              = "string"
+    description       = "Memorystore for Redis instance ID"
+    generate_resolver = true
+  }
+
+  column "alternative_location_id" {
+    description = "If specified, at least one node will be provisioned in this zone in addition to the zone specified in `location_id`"
+  }
+
+  column "auth_enabled" {
+    description = "Indicates whether OSS Redis AUTH is enabled for the instance"
+  }
+
+  column "authorized_network" {
+    description = "The full name of the Google Compute Engine network (https://cloud.google.com/vpc/docs/vpc) to which the instance is connected"
+  }
+
+  column "connect_mode" {
+    description = "The network connect mode of the Redis instance"
+  }
+
+  column "create_time" {
+    description = "The time the instance was created"
+  }
+
+  column "current_location_id" {
+    description = "The current zone where the Redis primary node is located"
+  }
+
+  column "customer_managed_key" {
+    description = "The KMS key reference that the customer provides when trying to create the instance"
+  }
+
+  column "display_name" {
+    description = "An arbitrary and optional user-provided name for the instance"
+  }
+
+  column "host" {
+    description = "Hostname or IP address of the exposed Redis endpoint used by clients to connect to the service"
+  }
+
+  column "labels" {
+    description = "Resource labels to represent user provided metadata"
+  }
+
+  column "location_id" {
+    description = "The zone where the instance will be provisioned"
+  }
+
+  column "maintenance_policy" {
+    description       = "The maintenance policy for the instance"
+    type              = "json"
+    generate_resolver = true
+  }
+
+  column "maintenance_schedule" {
+    description       = "Date and time of upcoming maintenance events which have been scheduled"
+    type              = "json"
+    generate_resolver = true
+  }
+
+  column "memory_size_gb" {
+    description = "Redis memory size in GiB"
+  }
+
+  column "name" {
+    description = "Unique name of the resource"
+  }
+
+  column "persistence_config" {
+    description       = "Persistence configuration parameters"
+    type              = "json"
+    generate_resolver = true
+  }
+
+  column "persistence_iam_identity" {
+    description = "Cloud IAM identity used by import / export operations to transfer data to/from Cloud Storage"
+  }
+
+  column "port" {
```

Comment:
Just curious, why was it necessary to include all these column descriptions here? Could they not be extracted from the struct?

Reply:
Because the default parser is the 1st in the list, and the descriptions in this strict ofter start with `Optional.` that makes the default parser to drop all other words.

IMO, the default parser that strips the description of the last dot and stuff should be the last one called.